### DISCUSSION TITLE
Welcome to BM Discussions!

### DISCUSSION THREAD

**Discussion by ComputerElite** [ComputerElite/BM#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BladedCargo** [comment]
Just want to say that this is under appreciated.

**Comment by Fire5818** [comment]
Why does this want access to my regedit its kinda sketch

**Reply by FranciscoRibeiro03** [reply]
To be able to register itself as an app that can open beatsaver:// links, which is the protocol used by OneClick™ Install

**Reply by ComputerElite** [reply]
If you don‘t want to give it Access to the registry that‘s totale Fine. You just won‘t be able to use OneClick install. As BMBF Manager creates .reg Files next to the exe you can Check them and run them yourself to enable OneClick install.

**Reply by Fire5818** [reply]
alr
